core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -92,6 +92,7 @@ class GroupCoordinatorTest {
   private val protocolSuperset = List((protocolName, metadata), ("roundrobin", metadata))
   private val requireStable = true
   private var groupPartitionId: Int = -1
+  val groupMetadataTopicId = Uuid.fromString("JaTH2JYK2ed2GzUapg8tgg")
```

Comment:
nit: any reason we choose this over a random uuid?

Reply:
It easier to reason about as we know what to expect, I also saw in other PR reviews some feedback suggesting using this instead of random uuid

Reply:
I'm not sure if I totally follow, but sure. If you have a reference to that conversation can you share it?

Reply:
I don't have the reference, bit hard to find comment in pr review. But @soarez mentioned he saw couple before. I don't really attached to either way.